lib/logstash/outputs/logstash.rb

```diff
@@ -99,6 +99,8 @@ def inner_http_output_options
         http_options['user'] = @username
         http_options['password'] = @password || fail(LogStash::ConfigurationError, '`password` is REQUIRED when `username` is provided')
         logger.warn("HTTP Basic Auth over non-secured connection") if @ssl_enabled == false
```

Comment:
Oops:

```suggestion
        logger.warn("transmitting credentials over non-secured connection") if @ssl_enabled == false
```

Reply:
And similar on the input plugin :facepalm::

~~~ diff
@@ -105,1 +105,1 @@
-        logger.warn("HTTP Basic Auth over non-secured connection") if @ssl_enabled == false
-        logger.warn("requiring credentials over non-secured connection") if @ssl_enabled == false
~~~

Reply:
🤦 (updated)